MetaPhase-Consulting/State-TalentMAP-API
# debugging

[Ticket](https://metaphase.atlassian.net/jira/software/c/projects/TM/boards/1?assignee=712020%3A3d9639a5-439c-4e2d-9bce-37b6924384d2&selectedIssue=TM-6873)

Description: Fixing issues with the XML attachment that are meant to be a part of the TM1 Memo's users can send out. 
NOTE - there is time variability with the SMTP server we are currently using so emails may reach your inbox later then expected.

To test follow these steps:

1. Go to Dashboard 
2. Select AO/CDO -> Employees and Panel Agendas
3.  Select 1st profile 
4. Profile's Assignment History 
5. Maintain Assignments
6. Select assignment with status "BR" 
7. Hit "Include" to include the assignment in the Memo
8. Scroll to the top of the page
9. Select "Add Memo"
10. Edit memo to include your state email under "To" section 
11. Scroll to the bottom of. the page and select "Select Memo Recipients" 
12. Look up your name in the GAL Lookup functionality 
13. Select your name (can also select multiple other recipients if wanted)
14. Click "Email Memo"
15. Check inbox for email


Example email with attachment:

<img width="1447" alt="SUCCESS Felicia's xml email " src="https://github.com/user-attachments/assets/a119b97e-b0c3-4b20-9e47-e98d00410740" />
